pair.c compiler warning

```
src/sp/protocol/pair1/pair.c:418:3: warning: 'l' may be used uninitialized in this function [-Wmaybe-uninitialized]
```
Not sure if it matters, though.

--- a/src/sp/protocol/pair1/pair.c
+++ b/src/sp/protocol/pair1/pair.c
@@ -374,7 +374,7 @@ pair1_send_sched(pair1_sock *s)
 	pair1_pipe *p;
 	nni_msg *   m;
 	nni_aio *   a = NULL;
-	size_t      l;
+	size_t      l = 0;
 
 	nni_mtx_lock(&s->mtx);
 

--- a/src/testing/marry.c
+++ b/src/testing/marry.c
@@ -1,5 +1,5 @@
 //
-// Copyright 2020 Staysail Systems, Inc. <info@staysail.tech>
+// Copyright 2021 Staysail Systems, Inc. <info@staysail.tech>
 // Copyright 2018 Capitar IT Group BV <info@capitar.com>
 //
 // This software is supplied under the terms of the MIT License, a
@@ -220,7 +220,7 @@ replace_port_zero(const char *addr, char *buf, int port)
 {
 	int  i;
 	int  j;
-	bool colon;
+	bool colon = false;
 	char c;
 
 	for (i = 0, j = 0; (c = addr[i]) != '\0'; i++) {

